Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -570,9 +570,9 @@ private IEnumerable<OrderTicket> GenerateOrders(OptionStrategy strategy, int str
 
                     if (optionSeq.Count() != 1)
                     {
-                        var error = string.Format("Couldn't find the option contract in algorithm securities list. Underlying: {0}, option {1}, strike {2}, expiration: {3}",
-                                strategy.Underlying.ToString(), optionLeg.Right.ToString(), optionLeg.Strike.ToString(), optionLeg.Expiration.ToString());
-                        throw new InvalidOperationException(error);
+                        throw new InvalidOperationException("Couldn't find the option contract in algorithm securities list. " +
+                            $"Underlying: {strategy.Underlying}, option {optionLeg.Right.ToString()}, strike {optionLeg.Strike.ToStringInvariant()}, " +
```

Comment:
`optionLeg.Right.ToString()` -- is this intentional or should be `ToStringInvariant()`?

Related question: 
I see `ToStringInvariant()` is used everywhere also for `enum` values, is this really required?
Wouldn't these be all the same?
1. `{optionLeg.Right.ToString()}`
2. `{optionLeg.Right.ToStringInvariant()}`
3. `{optionLeg.Right}`